web-console/src/lib/components/connectors/dialogs/UrlConnector.tsx

```diff
@@ -2,26 +2,28 @@
 
 import TabFooter from '$lib/components/connectors/dialogs/tabs/TabFooter'
 import TabLabel from '$lib/components/connectors/dialogs/tabs/TabLabel'
-import { ConnectorFormNewRequest, ConnectorFormUpdateRequest } from '$lib/services/connectors/dialogs/SubmitHandler'
 import { connectorTypeToConfig, connectorTypeToIcon, parseUrlSchema } from '$lib/functions/connectors'
-import { ConnectorType } from '$lib/types/connectors'
-import ConnectorDialogProps from '$lib/types/connectors/ConnectorDialogProps'
+import { ConnectorFormNewRequest, ConnectorFormUpdateRequest } from '$lib/services/connectors/dialogs/SubmitHandler'
```

Comment:
not something we have to do in this PR, but we should configure the imports to order automatically on save and check it as part of format check in CI, just so we don't always have a ton of re-ordering in commits